nipype/__init__.py

```diff
@@ -13,7 +13,7 @@
 """
 import os
 
-# XXX Deprecate this import
+# XXX Deprecated import. Not used internally but could be used externally.
```

Comment:
Nipype is in maintenance mode. No sense marking things as deprecated or to-deprecate at this point.
```suggestion
# No longer used internally but could be used externally.
```